pyscriptjs/src/exceptions.ts

```diff
@@ -0,0 +1,11 @@
+import { _createAlertBanner } from "./utils"
+
+
+export class UserError extends Error {
+  showBanner: boolean
+  constructor(message: string, showBanner = true) {
+    super(message)
+    this.name = "UserError"
+    this.showBanner = showBanner
+  }
+}
```

Comment:
do we still need `showBanner`? Again, if it's an error it should be showed, else it's not an error.
I can imagine that at some point we also want to configure "how, if and where" user errors are displayed, but this is something which is the responsibility of the caller, not the callee IMHO.

Reply:
Good question, I was thinking that in the future we might need, but I am happy to remove it and leave it for when we actually need the logic there

Reply:
yes, +1 to kill code-which-we-dont-need-now.
If it's not present, it cannot be buggy 😅